recipes/pdal/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
```

Comment:
Is it possible to test to make sure we can do what's described in the docs? https://pdal.io/development/cmake.html#basic-cmake-configuration

Reply:
Modified test to use find_package in config mode.

To maintain even closer CMakeLists.txt to the upstream example, we can switch to `find_package` in module mode and old-style global linking (see [gist](https://gist.github.com/16401a7e4e8ddc4b1a7e21657a11f9da)) but I suppose target mode is preferred. What do you think?

Reply:
I like to have assurances as a consumer, having the test package match (closely) upstream is ideal in my world.

I love the patch you linked against :100:

Reply:
Applied the patch. The build goes fine.